nerdlandbot/helpers/constants.py

```diff
@@ -47,3 +53,6 @@
 DAD_JOKE_URL = 'https://icanhazdadjoke.com'
 EIGHT_BALL_URL = 'https://evening-castle-06130.herokuapp.com/eightball'
 
+#Nerdland server
+DISCORD_SERVER_ID = 712743925114339328 #bot test server for testing purposes
```

Comment:
Very well done! One minor change that would be required (I think?):

Should this not be an environment variable (from .env), rather than a constant?

Reply:
My reasoning was that the env file is for access tokens and primary info for the bot. I do not consider this primary info since it is only relevant for one sepcific feature, so I made it a constant.

Why would it be better to make it an environment variable?

Reply:
There's definitely nothing wrong with doing it this way. Currently we've got all things that are different between testbot and prodbot in the .env file. In constants.py we (currently) only have things that don't need to be configured.

I'm ok with keeping it this way, which is why I made it a normal comment without blocking the PR :)